tests/test_repocard.py

```diff
@@ -130,6 +130,11 @@
 ---
 """
 
+DUMMY_MODELCARD_NO_TEXT_CONTENT = """---
+license: cc-by-sa-4.0
```

Comment:
would expect this to fail on missing language (but maybe the spec has been updated to not have it be required?)

Reply:
I don't know to be honest. I took the example from the reported issue to be sure I'm testing what was reported.